llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -174,6 +174,17 @@ static cl::opt<std::string> WorkloadDefinitions(
              "}"),
     cl::Hidden);
 
+static cl::opt<bool> ImportAssumeUniqueLocal(
```

Comment:
Unrelated change should be removed

Reply:
this is from #100448?

Reply:
Oh sorry, I was looking at diffs from my last review, and wasn't expecting things from other patches to show up (did it get rebased?)

Reply:
Yes, I needed to so I could get the llvm-ctxprof-util tool